CromIAM/src/test/scala/cromiam/cromwell/CromwellClientSpec.scala

```diff
@@ -0,0 +1,108 @@
+package cromiam.cromwell
+
+import java.net.URL
+
+import akka.actor.ActorSystem
+import akka.event.NoLogging
+import akka.http.scaladsl.model.HttpHeader
+import akka.http.scaladsl.model.headers.{Authorization, OAuth2BearerToken}
+import akka.stream.ActorMaterializer
+import cromiam.auth.User
+import cromwell.api.model.{WorkflowId, WorkflowLabels, WorkflowMetadata}
+import org.scalatest.{AsyncFlatSpec, BeforeAndAfterAll, Matchers}
+import cromwell.api.{CromwellClient => CromwellApiClient}
+import org.broadinstitute.dsde.workbench.model.WorkbenchUserId
+import spray.json.{JsObject, JsString}
+
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutor, Future}
+
+class CromwellClientSpec extends AsyncFlatSpec with Matchers with BeforeAndAfterAll {
+  import CromwellClientSpec._
+
+  import ExecutionContext.Implicits.global
+
+  implicit val actorSystem: ActorSystem = ActorSystem("CromwellClientSpec")
+  implicit val ece: ExecutionContextExecutor = actorSystem.dispatcher
+  implicit val materializer: ActorMaterializer = ActorMaterializer()
+
+  val cromwellClient = new MockCromwellClient()
+
+  override protected def afterAll(): Unit = {
+    actorSystem.terminate()
+    super.afterAll()
+  }
+
+  "CromwellClient" should "eventually return a subworkflow's root workflow id" in {
+    cromwellClient.getRootWorkflow(SubworkflowId.id.toString, FictitousUser).map(w => assert(w == RootWorkflowId.id.toString))
```

Comment:
i don't know about the user but the correctness of the spelling is certainly fictitious

Reply:
Who are you to tell this user how to spell their name?